package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-double-marquee",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
Sorry for missing this -- we can keep the version, it will get bumped automatically

Reply:
fix'd